Products/ZenHub/services/DiscoverService.py

```diff
@@ -303,3 +303,24 @@ def remote_moveDevice(self, dev, path):
     def remote_getDefaultNetworks(self):
         monitor = self.dmd.Monitors.Performance._getOb(self.instance)
         return [net for net in monitor.discoveryNetworks]
+
+    @translateError
+    def remote_removeNetAppInterfaces(self, net):
+        """
+        Remove IPs for particular network
+        already assigned to NetApp interfaces (nodes)
+
+        @param net - network to discover
+        @return:  a list of IPs without IP assigned for NetApp interfaces
+        @rtype: list
+        """
+
+        full_ip_list = net.fullIpList()
+
+        for d in self.dmd.Devices.getSubDevices():
```

Comment:
why not do dmd.Devices.Storage.NetApp.getSubDevices()?  would be much more efficient.